app/Image/StreamStatFilter.php

```diff
@@ -26,6 +26,7 @@ public function filter($in, $out, &$consumed, bool $closing): int
 	{
 		while ($bucket = stream_bucket_make_writeable($in)) {
 			$consumed += $bucket->datalen;
+
```

Comment:
```suggestion
```

So that this file is removed from the diff.

Reply:
Fixed [30ecdef](https://github.com/LycheeOrg/Lychee/pull/1469/commits/30ecdef753c7135dc6d34b141e672ef563eaba5e)